.github/labels.md

```diff
@@ -182,10 +182,23 @@ with `@`-mentions if someones or some groups special knowledge is required.
 
 #### Label `status:invalid`
 
-The Issue contains content that fails to address its intention.  An example
-would be a feature request wasn't discussed before in the forums or for Pull
-Requests those requests that compete to address the same topic, but were not
-chosen to be merged.  Those Issues shouldn't be merged.
+The Issue doesn't classified as formal valid Issue report or applicable Pull
+Request.  Reasons for this can be manifold:
```

Comment:
> manifold -> many-fold ?

See http://dictionary.cambridge.org/dictionary/english/manifold

But yes, I can use the rephrased version.  Manifold seems to be used almost only in engineering and math in english.